kit/src/main/java/com/oracle/javafx/scenebuilder/kit/metadata/Metadata.java

```diff
@@ -4470,7 +4472,12 @@ public boolean isPropertyTrimmingNeeded(PropertyName name) {
                 com.oracle.javafx.scenebuilder.kit.metadata.property.value.DoublePropertyMetadata.DoubleKind.COORDINATE,
                 true, /* readWrite */
                 100.0, /* defaultValue */
-                new InspectorPath("Properties", "Specific", 98));
+                new InspectorPath("Properties", "Specific", 10));
```

Comment:
Why did you change 98 here?

Reply:
This was an accident. This file was kind of one of the hardest files to edit, looks all the same, IDE gets sluggish. Sorry for that.